tools/Dashboard/DevHome.Dashboard/Controls/WidgetControl.xaml.cs

```diff
@@ -17,17 +21,33 @@
 using Microsoft.UI.Xaml.Controls;
 using Microsoft.Windows.Widgets;
 using Serilog;
+using Windows.UI.ViewManagement;
+using WinUIEx;
 
 namespace DevHome.Dashboard.Controls;
 
+[ObservableObject]
 public sealed partial class WidgetControl : UserControl
 {
     private readonly ILogger _log = Log.ForContext("SourceContext", nameof(WidgetControl));
 
+    private readonly UISettings _uiSettings = new();
+
     private readonly StringResource _stringResource;
 
+    private readonly double _headerHeightUnscaled = 36;
+
     private SelectableMenuFlyoutItem _currentSelectedSize;
 
+    [ObservableProperty]
+    private GridLength _headerHeight;

```

Comment:
Technically, these attributes should only be in ViewModels because the view should have no logic besides showing the view.

However, these members control the view and make the view simpler to understand.

I don't know when too much spillover of `[ObservableProperty]` into the view is too much.

But.  I'm not blocking on this.  Just a though.